cmd/ausoceantv/stripe.go

```diff
@@ -129,3 +175,102 @@ func (svc *service) handleCreatePaymentIntent(c *fiber.Ctx) error {
 
 	return c.JSON(v)
 }
+
+func (svc *service) createSubscriptionIntent(c *fiber.Ctx, cid, pid string) error {
+	// Set the subscription parameters.
+	paymentSettings := &stripe.SubscriptionPaymentSettingsParams{
+		SaveDefaultPaymentMethod: stripe.String("on_subscription"),
+	}
+	subParams := &stripe.SubscriptionParams{
+		Customer: stripe.String(cid),
+		Items: []*stripe.SubscriptionItemsParams{{
+			Price: stripe.String(pid),
+		}},
+		PaymentSettings: paymentSettings,
+		PaymentBehavior: stripe.String("default_incomplete"),
+	}
+	subParams.AddExpand("latest_invoice.payment_intent")
+	s, err := subscription.New(subParams)
+	if err != nil {
+		log.Errorf("subscription.New: %v", err)
```

Comment:
Is this log redundant if we're just going to log the return error below on the level above ?